app/tests/test_download.py

```diff
@@ -0,0 +1,136 @@
+import boto3
+from moto import mock_s3
+
+from app.tests.factories import (
+    ConsignmentFactory,
+    FileFactory,
+    FileMetadataFactory,
+)
+
+BUCKET = "test-download"
+CONSIGNMENT_REF = "TDR-123-TEST"
+FILE_PATH = "data/content/folder_a/test_file.txt"
+FILE_NAME = "test_file.txt"
+
+
+def mock_record():
+    """
+    Creates a dummy record to be used by tests
+    """
+    consignment = ConsignmentFactory(ConsignmentReference=CONSIGNMENT_REF)
+    file = FileFactory(
+        file_consignments=consignment,
+        FileName=FILE_NAME,
+        FilePath=FILE_PATH,
+        FileType="file",
+    )
+
+    metadata = {
```

Comment:
I think this test can get away without creating the file metadata objects

Reply:
I tried it 
The function requires metadata 
`    s3 = boto3.client("s3")
    bucket = current_app.config["RECORD_BUCKET_NAME"]
    file_metadata = get_file_metadata(record_id)`